onadata/libs/tests/utils/test_export_tools.py

```diff
@@ -349,6 +403,49 @@ def test_should_not_create_new_export_when_old_exists(self):
             Export.objects.get(xform=self.xform).options,
         )
 
+        # New metadata will yield a new export
+        metadata = MetaData.objects.create(
+            content_type=ContentType.objects.get_for_model(XForm),
+            data_type="media",
+            data_value=f"xform_geojson {self.xform.id} testgeojson2",
+            extra_data={
+                "data_title": "end",
+                "data_fields": "",
+                "data_geo_field": "qn09",
+                "data_simple_style": True,
+            },
+            object_id=self.xform.id,
+        )
+        _response = custom_response_handler(
```

Comment:
Looks like the variable `_response` is unused

Reply:
In that case, the convention is usually to use an `_` underscore as the variable name. Though in this case I think we can just ignore the result from the function call and not assign it to a variable